src/webgl/camera.js

```diff
@@ -275,36 +275,43 @@ p5.RendererGL.prototype.perspective = function(fovy, aspect, near, far) {
  * 3 3d boxes, reveal several more boxes on 3d plane when mouse used to toggle
  *
  */
-p5.prototype.ortho = function(left, right, bottom, top, near, far) {
-  left = left || -this.width / 2;
-  right = right || this.width / 2;
-  bottom = bottom || -this.height / 2;
-  top = top || this.height / 2;
-  near = near || 0;
-  far = far || Math.max(this.width, this.height);
-  this._renderer.uPMatrix = p5.Matrix.identity();
-  //this._renderer.uPMatrix.ortho(left,right,bottom,top,near,far);
 
-  var w = right - left;
-  var h = top - bottom;
-  var d = far - near;
+p5.prototype.ortho = function(){
+  this._renderer.ortho.apply(this._renderer, arguments);
+  return this;
+};
 
-  var x = 2.0 / w;
-  var y = 2.0 / h;
-  var z = -2.0 / d;
+p5.RendererGL.prototype.ortho = function(l, r, b, t, n, f) {
+  // ortho() ... top/left = 0/0
+  if(l === undefined) l = 0;
```

Comment:
thank you for this fix! I'm noticing that these values were also changed from the former defaults (-this.width/2, for example, for left)—can you expand on the reasoning for this? in GL mode 0,0 is in the center of the canvas.

I would also prefer if we can keep the expanded variable names for left, right, bottom, top, etc., which I think makes the code more readable.